addons/project/models/project.py

```diff
@@ -1888,6 +1889,11 @@ def _load_records_create(self, vals_list):
                 else:
                     for field_name in self._get_recurrence_fields() + ['recurring_task']:
                         vals.pop(field_name, None)
+            project_id = vals.get('project_id')
+            if project_id:
+                project = self.env['project.project'].browse(project_id)
+                default_val = self.with_context(default_project_id=project.id).default_get(['project_id'])
+                vals.update(**default_val)
```

Comment:
Why do we need this code?

Reply:
I guess to get the AA and analytic tags set on the project. 🙂